src/main/java/org/graylog/plugins/collector/CollectorModule.java

```diff
@@ -43,5 +45,7 @@ protected void configure() {
         addPermissions(CollectorRestPermissions.class);
 
         addAuditEventTypes(CollectorAuditEventTypes.class);
+
+        serviceBinder().addBinding().to(ConfigurationEtagService.class).in(Scopes.SINGLETON);
```

Comment:
We see that on each request a new and empty cache instance is generated. Any idea why? Even when bind as singleton?